.github/workflows/ci.yml

```diff
@@ -5,7 +5,21 @@ on:
   pull_request:
 
 jobs:
+  test-repository:
```

Comment:
```suggestion
  test-project:
```

Reply:
This is testing only the "repository" workspace (component).